docs/paying-for-lit/delegating-credit.md

```diff
@@ -0,0 +1 @@
+# Delegating Credits
```

Comment:
Do we need a general "delegating credits" section in addition to the payment delegation DB section? Wondering if we can just consolidate somehow so this whole thing is quicker to read through.

Reply:
these technically are two separate paths though right? Like they can delegate using the SDK or the DB solution right?

Reply:
but yea, maybe we could list these on the same page under two different headers ('using the SDK' and 'using the DB')